README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    - semantic HTML is using tags that describe what is within. Instead of using just "div", use "dive class = '' "
+
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
 
+    - Display Block is changing the items to diplay in a block manner so that the next item appears on the next line down; Inline is letting the next item appear on the same line.
```

Comment:
great definition, many people struggle with this concept but you have a solid understanding